docs/agent/hostname-resolution.md

```diff
@@ -10,6 +14,12 @@ On upgrades from Agent v5 to Agent v6 (<6.6.0), this may make the Agent hostname
 
 Starting from the Agent v6.3 a configuration flag called `hostname_fqdn` has been introduced that allows the Agent v6 to have the same behavior as Agent v5. This flag is disabled by default on version 6.3 and enabled by default in version 6.6.
 
+## Windows
+
+On Agent v5, the ageent Windows reported the unqualified hostname by default.  To maintain backward compatibility, this behavior is preserved with Agent v6.  The new flag `hostname_fqdn` will remain disabled by default on Windows.
+
+The Windows Agent v6 will honr the configuration flag.  Setting `hostname_fqdn` to true will result in the Windows agent reporting the fully qualified hostname.
```

Comment:
Only Agent >= v6.5 actually honor the config flag, let's clarify this.

Also, typo: `honor`